### PR TITLE
[WIP] Add account section

### PR DESCRIPTION
**Pull request by stephaurelio**
### Proposed changes

This PR adds a section to the docs for user accounts.

### Related issues (optional)

Closes JIRA: https://docker.atlassian.net/browse/ENGDOCS-1794

### CODE REVIEW

File: data/toc.yaml [hunk 1]
```diff
@@ -2053,6 +2051,15 @@ Manuals:
   - path: /docker-hub/release-notes/
     title: Release notes
 
+- sectiontitle: Docker account
+  section:
+  - path: /docker-account/
```
Comment (dvdksn) [inline data/toc.yaml:2056]: how about `accounts` for the section slug? i.e. `https//docs.docker.com/accounts/manage-account`

**Comment by dvdksn** [comment]
@stephaurelio added a new label and config for the labeler action to automatically apply the area/accounts label to pages in the new section

File: content/accounts/manage-account.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+> **Tip**
+>
+> If your account is associated with an organization that enforces Single Sign-On (SSO), you may not have permissions to update your account settings. You need to contact your administrator to update your settings.
+{ .tip }
+
+## Update general settings
+
+1. Sign in to your [Docker account](https://account.docker.com/login).
+2. In Docker Home, select your avatar in the top-right corner to open the dropdown.
+3. Select **My Account**.
```
Comment (pla-sam) [inline content/accounts/manage-account.md:18]: I think this would be `Account Settings`, not `My Account`.

Comment (stephaurelio) [reply content/accounts/manage-account.md:18]: Will that be changing soon? I still see **My Account** in stage.

Comment (pla-sam) [reply content/accounts/manage-account.md:25]: Maybe it is `My Account`. Sorry!

Comment (stephaurelio) [reply content/accounts/manage-account.md:25]: Maybe not for long 😅

**Comment by stephaurelio** [comment]
New PR here: https://github.com/docker/docs/pull/20350

**Pull request closed without merging**